### PR TITLE
fix: user init mail (for wrong email)

### PR DESCRIPTION
**Pull request by livio-a**

### CODE REVIEW

**Review by fgerschwiler — APPROVED** [review]

File: console/src/app/pages/users/user-detail/auth-user-detail/resend-email-dialog/resend-email-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+</h1>
+<p class="desc">{{'USER.SENDEMAILDIALOG.DESCRIPTION' | translate}}</p>
+<div mat-dialog-content>
+    <mat-form-field class="formfield">
+        <mat-label>{{ 'USER.SENDEMAILDIALOG.NEWEMAIL' | translate }}</mat-label>
+        <input matInput [(ngModel)]="email" />
+    </mat-form-field>
+</div>
+<div mat-dialog-actions class="action">
+    <button cdkFocusInitial color="primary" mat-button class="ok-button" (click)="closeDialog()">
```
Comment (fgerschwiler) [inline console/src/app/pages/users/user-detail/auth-user-detail/resend-email-dialog/resend-email-dialog.component.html:12]: should'n this be secondary?

File: console/src/assets/i18n/de.json [hunk 2]
```diff
@@ -112,6 +114,11 @@
             "DELETE_TITLE":"User löschen",
             "DELETE_DESCRIPTION":"Sie sind im Begriff einen Benutzer endgültig zu löschen. Wollen Sie dies wirklich tun?"
         },
+        "SENDEMAILDIALOG":{
+            "TITLE":"Email Benachrichtigung senden",
+            "DESCRIPTION":"Klicken Sie den untenstehenden Button and die aktuelle Emailadresse zu versenden oder ändern Sie die Emailadresse in dem Feld.",
```
Comment (fgerschwiler) [inline console/src/assets/i18n/de.json:119]: Something is wrong with this message, should it be something like this:

```suggestion
            "DESCRIPTION":"Klicken Sie den untenstehenden Button um ein verifizierungs Email an die aktuelle Adresse zu versenden oder ändern Sie die Emailadresse in dem Feld.",
```

File: console/src/app/pages/users/user-detail/auth-user-detail/resend-email-dialog/resend-email-dialog.component.html (hunk 1 above)
Comment (peintnermax) [reply console/src/app/pages/users/user-detail/auth-user-detail/resend-email-dialog/resend-email-dialog.component.html:12]: youre right. ive removed initial focus on cancel

**Pull request merged**